x/incentives/client/cli/query.go

```diff
@@ -384,31 +386,80 @@ $ %s query incentives rewards-estimation
 				return err
 			}
 
-			lockIdStrs := strings.Split(lockIdsCombined, ",")
-			lockIds := []uint64{}
-			for _, lockIdStr := range lockIdStrs {
-				lockId, err := strconv.ParseUint(lockIdStr, 10, 64)
+			var res *lockuptypes.AccountLockedLongerDurationResponse
+			if owner != "" {
+				queryClientLockup := lockuptypes.NewQueryClient(clientCtx)
+
+				res, err = queryClientLockup.AccountLockedLongerDuration(cmd.Context(), &lockuptypes.AccountLockedLongerDurationRequest{Owner: owner, Duration: time.Millisecond})
 				if err != nil {
 					return err
 				}
-				lockIds = append(lockIds, lockId)
+			} else {
+				owner = ""
+			}
+
+			ownerLocks := []uint64{}
+
+			if res != nil {
```

Comment:
Why would `res` be nil if we already checked that there is no error above?

Reply:
Ahh good call, my bad! No need to put this within an "if" statement since we already checked above

Reply:
Hey, I made this suggestion and the cli_test started to fail because of it. Did I misinterpret your suggestion?